api/server.py

```diff
@@ -0,0 +1,61 @@
+import csv
+import json
+from os import path
+from flask import Flask
+from flask_restful import Resource, Api, abort, reqparse
+
+app = Flask(__name__)
+api = Api(app)
+
+parser = reqparse.RequestParser()
+parser.add_argument('limit', type=int, required=False,
+                    location='args')
+parser.add_argument('offset', type=int, required=False, location='args')
+
+
+def get_path(filename):
+    return 'csv/' + filename + '.csv'
+
+
+def csv_exists(filename):
+    return path.isfile(get_path(filename))
+
+
+def read_csv(filename, limit=-1, offset=0):
+    limit = limit or -1
```

Comment:
This will be somewhat of a nit-pick: -1 and 0may be extracted into variables.
Also, you're providing both with default values in the function signature, so setting default values for them shouldn't be necessary here.